modules/mediaforceBidAdapter.js

```diff
@@ -140,31 +150,47 @@ export const spec = {
           default: return;
         }
       }
-      validImp && imp.push(impObj);
 
-      let request = {
-        id: bid.transactionId,
-        site: {
-          page: referer,
-          ref: referer,
-          id: bid.params.publisher_id,
-          publisher: {
-            id: bid.params.publisher_id
+      let request = requestsMap[bid.params.publisher_id];
```

Comment:
Curious why you are keying against the publisher_id and sending multiple requests if the goal of the ticket is to allow for multiple bids in one request? It does not seem to change the endpoint url or any other attributes of the request.

Couldn't you organize your imp object by `params.publisher_id` and send that in one request?

Reply:
As i wrote upper, some bids in theory can have different publisher_id in params, and because the publisher_id defines site field, which is common to all imps, we must prepare separate request for each publisher_id.

Reply:
Could you explain a situation where one Prebid auction can have multiple sites or publishers?

Reply:
I think not, but just in case, I would like to leave this opportunity